src/server/pkg/obj/obj.go

```diff
@@ -132,7 +132,7 @@ type AmazonAdvancedConfiguration struct {
 	// uploader, and not the owner of the bucket. Using the default ensures that
 	// the owner of the bucket can access the objects as well.
 	UploadACL      string `env:"UPLOAD_ACL, default=bucket-owner-full-control"`
-	Reverse        bool   `env:"REVERSE, default=true"`
+	Reverse        bool   `env:"REVERSE, default=false"`
```

Comment:
I think we want to keep this as default=true as lame as that is. Because otherwise existing user's clusters will break when they upgrade and use the same `pachctl deploy` command that they used in the previous version.

Reply:
I don't see why we would keep this as `default=true` - the old default was never used until the fix in `cmdutil/env.go` in this PR.  The old default in practice was `false`.

Reply:
Interesting I didn't realize this was false in practice, that's actually kind of convenient.